AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_MaterialAdded.py

```diff
@@ -4,6 +4,7 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
+import os
```

Comment:
should this move down with the other imports in AtomEditorComponents_Material_AddedToEntity